lib/verbalize/action.rb

```diff
@@ -50,6 +42,14 @@ def call!
 
       private
 
+      def input(*required_keywords, optional: [])
+        @required_inputs = required_keywords
+        optional = Array(optional)
+        @optional_inputs = optional
+
+        class_eval Build.call(required_keywords, optional)
+      end
```

Comment:
No changes, only privatized. So you can still call input from inside the class, e.g.
```ruby
1] pry(main)> class Add
[1] pry(main)*   include Verbalize::Action
[1] pry(main)*
[1] pry(main)*   input :a, :b
[1] pry(main)*
[1] pry(main)*   def call
[1] pry(main)*     a + b
[1] pry(main)*   end
[1] pry(main)* end
=> :call
[2] pry(main)> Add.input
NoMethodError: private method `input' called for Add:Class
from (pry):10:in `__pry__'
[3] pry(main)> Add.call!(a: 1, b: 1)
=> 2
```